tools/src/test/resources/1.6/valid-component-identifiers-1.6.json

```diff
@@ -11,8 +11,11 @@
       "version": "1.0.0",
       "cpe": "cpe:2.3:a:example:acme-library:1.0.0:*:*:*:*:*:*:*",
       "purl": "pkg:maven/com.example/acme-library@1.0.0",
-      "omnibor": "gitoid:blob:sha1:261eeb9e9f8b2b4b0d119366dda99c6fd7d35c64",
-      "swhid": "swh:1:cnt:94a9ed024d3859793618152ea559a168bbcbb5e2"
+      "omnibor": [
+        "gitoid:blob:sha1:261eeb9e9f8b2b4b0d119366dda99c6fd7d35c64",
```

Comment:
Thank you. I think to make the omnibor information complete the Artifact Dependency Graph must be represented in `dependencies` section using `provides`.

https://omnibor.io/glossary/artifact_dependency_graph/